freqtrade/commands/hyperopt_commands.py

```diff
@@ -63,8 +71,14 @@ def start_hyperopt_show(args: Dict[str, Any]) -> None:
 
     config = setup_utils_configuration(args, RunMode.UTIL_NO_EXCHANGE)
 
-    only_best = config.get('hyperopt_list_best', False)
-    only_profitable = config.get('hyperopt_list_profitable', False)
+    filteroptions = {
+        'only_best': config.get('hyperopt_list_best', False),
+        'only_profitable': config.get('hyperopt_list_profitable', False),
+        'filter_min_avg_time': config.get('hyperopt_list_min_avg_time', 0),
+        'filter_max_avg_time': config.get('hyperopt_list_max_avg_time', 0),
+        'filter_min_avg_profit': config.get('hyperopt_list_min_avg_profit', 0),
+        'filter_min_total_profit': config.get('hyperopt_list_min_total_profit', 0)
```

Comment:
better use None as default for `get()` here (for all 4 new options) -- this would allow checking against negative profits, selecting `--min-avg-profit -0.001`, for example...

Reply:
fixed in latest commit and working with -proffits